todo_app/templates/index.html

```diff
@@ -24,12 +24,19 @@ <h2>Add Item</h2>
 
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h2>To-do List</h2>
       <ul class="list-group mb-4">
           <ul>
             <li>
               {% for i in list1 %}
                 <h2>{{ i }}</h2>
```

Comment:
Currently this output looks fairly ugly, and it will get worse if we try print the `Item` objects in full!
Breaking this down to say what we actually want to display will help considerably, maybe just:
```suggestion
                <h2>{{ i.name }} - {{ i.status }}</h2>
```